### ISSUE TITLE
Review data portal Web site content

### ISSUE THREAD

**Issue by rdgoite** [HumanCellAtlas/ingest-central#209]
- [ ] Review [about page section for Ingest Service](https://dev.data.humancellatlas.org/about/what-is-the-platform/ingest-service/)
- [ ] Review [Ingest Broker development guide](https://dev.data.humancellatlas.org/develop/development-guides/ingest-broker-development-guide)

**Comment by justincc** [comment]
Is this relevant any more? Neither of the pages linked exist.

**Comment by lauraclarke** [comment]
I don't think this is relevant but we do need a view on what information about the ingest APIs should exist on the main DCP data portal. This can probably be moved to a strategic conversation though

**Comment by malloryfreeberg** [comment]
Perhaps with the Upload Service, but there should be some documentation about what metadata and data validation happens (thinking from the perspective of a data contributor).

**Comment by lauraclarke** [comment]
I was struggling to find any documentation about contribution on the data portal right now, @malloryfreeberg do you know where it lives?

**Comment by malloryfreeberg** [comment]
[Here](https://prod.data.humancellatlas.org/contribute) is the old content.

I have an [open PR](https://github.com/HumanCellAtlas/data-portal-content/pull/278) for updates to this section that includes the addition of the Dataset Suitability info. The contribute section includes describing what kind of data can be submitted, what contributors need to prepare, and what happens to the data after it's in the DCP (e.g. processing, browsing, matrix service).
